fe1-web/src/__tests__/utils/TestUtils.ts

```diff
@@ -39,6 +43,28 @@ export const mockLaoState: LaoState = {
 };
 export const mockLao = Lao.fromState(mockLaoState);
 
+// MOCK ROLL CALL
+export const mockId = new Hash('rollCallId');
+export const mockName = 'myRollCall';
+export const mockLocation = 'location';
+export const mockTimestampStart = new Timestamp(1620255600);
+export const mockTimestampEnd = new Timestamp(1620357600);
+export const mockAttendees = ['attendee1', 'attendee2'];
+export const mockRollCallState: any = {
+  id: mockId.valueOf(),
+  eventType: EventTypeRollCall,
+  start: mockTimestampStart.valueOf(),
+  name: mockName,
+  location: mockLocation,
+  creation: mockTimestampStart.valueOf(),
+  proposedStart: mockTimestampStart.valueOf(),
+  proposedEnd: mockTimestampEnd.valueOf(),
+  status: RollCallStatus.CLOSED,
+  attendees: mockAttendees,
+};
+
+export const mockRollCall = RollCall.fromState(mockRollCallState);
```

Comment:
Do you really need all of these to be exported ? Try to export only what you will need elsewhere. If you realize that it is used in only one test file, you'd better define it in it directly. 
In addition, I would rename these variables to make clear that they are all linked to roll call tests (for example, `mockId` and `mockName` are clearly related to roll calls if we see what is in there, but their name doesn't reflect this).